packages/yoga/src/Input/web/Input.jsx

```diff
@@ -98,6 +105,8 @@ const Input = React.forwardRef(
       rightIcon,
       a11yId,
       includeAriaAttributes,
+      prependElement,
```

Comment:
WDYT of naming this `leftElement` or `leftContent`?

Reply:
Makes sense, so it will use the simular nomenclature from the right positioned icon.